### ISSUE TITLE
Xlsx not working on bigger reports

### ISSUE THREAD

**Issue by pofider** [jsreport/jsreport-worker#1]
See this on https://playground.jsreport.net/w/jan_blaha/GndyMeVG

The issue is that the xlsx recipe writes for performance reason files to the temp directly from helpers (script manager). These file are afterwards read from the recipe.
The old jsreportonline solves this using exrtra code that reads those files in `tasks` container, encode it to base64 and decode it back in the `main` container.

https://github.com/jsreportonline/tasks/blob/master/index.js#L11
https://github.com/jsreportonline/main/blob/master/lib/xlsx.js#L7

The new worker container should handle this somehow. Maybe worker should read those files and send it as base64. The same way we do in the current tasks container. The docker manager would read those files and write to temp files so the xlsx doesn't need to be changed. And the performance reasons are satisfied.

In other words we need the same code as in old jsreportonline to run in the new one.

**Comment by bjrmatos** [comment]
in fact the error was related to some options having paths that use temp directory that does not exists inside worker, [the solution](https://github.com/jsreport/jsreport-worker/commit/7288f84beae3096a1bfe96d081f3674f36cc4283) was to use the local temp directory defined [here](https://github.com/jsreport/jsreport-worker/blob/master/lib/reporter.js#L21) (a path that exists).

i think the solution should be fine since theoretically a single render pipeline should always end up in same worker, however if we have other concerns we can add the mentioned workaround code and transfer those files as base64 always in the case that for some reason something in the pipeline ends with a different worker than the one that generated the files

**Comment by bjrmatos** [comment]
fixed in 7288f84beae3096a1bfe96d081f3674f36cc4283
